src/fourcipp/utils/cli.py

```diff
@@ -23,9 +23,11 @@
 
 import argparse
 import sys
+from pathlib import Path
```

Comment:
We generally import in this form (besides fourcipp packages and loguru)
```suggestion
import pathlib
```